daemon/container_operations_unix.go

```diff
@@ -120,19 +122,21 @@ func (daemon *Daemon) setupIpcDirs(c *container.Container) error {
 				return err
 			}
 
-			shmSize := int64(daemon.configStore.ShmSize)
```

Comment:
As the commit message says, this code is redundant (not needed). c.HostConfig.ShmSize has already been set to daemon default value, it happens in daemon.adaptContainerSettings, so here we can just use c.HostConfig.ShmSize directly.